contracts/interfaces/ITub.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity ^0.4.19;
+
+import "./DSValue.sol";
+import "./DSToken.sol";
+import "./ILiquidator.sol";
+
+interface ITub {
```

Comment:
I think this can go for all the Maker interfaces, but I think we should have Github links or something to make it easy to go and browse the code.